modules/sr/robot/motor_devices.py

```diff
@@ -23,10 +29,18 @@ def __init__(self, webot: Robot, motor_name: str) -> None:
         super().__init__(webot, motor_name)
         self.webot_motor.setPosition(float('inf'))
         self.webot_motor.setVelocity(0)
+        self.max_torque = self.webot_motor.getMaxTorque()
 
     def set_speed(self, speed: float) -> None:
         self.webot_motor.setVelocity(speed)
 
+        # If not using braking, set torque to 0.1 when motor speed is 0
+        # Otherwise set max torque
+        if(speed == 0 and not self._use_brake):
```

Comment:
```suggestion
        if speed == 0 and not self._use_brake:
```

Brackets not needed here :)

Reply:
Habits from other languages, will fix 👍